src/renderer/index.html

```diff
@@ -9,43 +9,127 @@
+        <!-- Activity Bar (VSCode-like narrow sidebar) -->
+        <aside class="activity-bar" id="activity-bar">
+            <div class="activity-bar-header">
+                <div class="app-icon" title="PowerPlatform ToolBox">PP</div>
+            </div>
+            <nav class="activity-bar-nav">
+                <button class="activity-item active" data-sidebar="tools" title="Installed Tools">
+                    <span class="activity-icon">🔧</span>
+                    <span class="activity-label">Tools</span>
                 </button>
-                <button class="nav-item" data-view="connections">
-                    <span class="nav-icon">🔗</span>
-                    <span class="nav-label">Connections</span>
+                <button class="activity-item" data-sidebar="connections" title="Connection Manager">
+                    <span class="activity-icon">🔗</span>
+                    <span class="activity-label">Connections</span>
                 </button>
-                <button class="nav-item" data-view="settings">
-                    <span class="nav-icon">⚙️</span>
-                    <span class="nav-label">Settings</span>
+                <button class="activity-item" data-sidebar="marketplace" title="Tools Marketplace">
+                    <span class="activity-icon">🛒</span>
+                    <span class="activity-label">Marketplace</span>
                 </button>
             </nav>
+            <nav class="activity-bar-footer">
+                <button class="activity-item" data-sidebar="settings" title="Settings">
+                    <span class="activity-icon">⚙️</span>
+                    <span class="activity-label">Settings</span>
+                </button>
+            </nav>
+        </aside>
+
+        <!-- Sidebar (context-sensitive based on Activity Bar selection) -->
+        <aside class="sidebar" id="sidebar">
+            <!-- Tools Sidebar -->
+            <div class="sidebar-content active" id="sidebar-tools">
+                <div class="sidebar-header">
+                    <h2 class="sidebar-title">Installed Tools</h2>
+                </div>
+                <div class="sidebar-body">
+                    <div class="tools-list" id="sidebar-tools-list">
+                        <!-- Tools will be populated here -->
+                    </div>
+                </div>
+            </div>
+
+            <!-- Connections Sidebar -->
+            <div class="sidebar-content" id="sidebar-connections">
+                <div class="sidebar-header">
+                    <h2 class="sidebar-title">Connection Manager</h2>
+                    <button class="btn btn-primary btn-sm" id="sidebar-add-connection-btn">+ Add</button>
+                </div>
+                <div class="sidebar-body">
+                    <div class="connections-list-sidebar" id="sidebar-connections-list">
+                        <!-- Connections will be populated here -->
```

Comment:
Add a pill on the top right corner of the connection card showing the environment type